portal-frontend/src/app/features/applications/edit-submission/parcel-details/parcel-entry/parcel-entry.component.html

```diff
@@ -321,7 +322,7 @@ <h6 class="owner-search-option" *ngIf="filteredOwners.length">OR</h6>
           </mat-radio-button>
         </mat-radio-group>
 
-        <div class="warning-banner" *ngIf="selectedOwner">
+        <div id="owner-info" class="warning-banner" *ngIf="selectedOwner">
```

Comment:
Since this component can exist multiple times per page, you cannot use an ID. IDs must be unique in the DOM.

Reply:
Good call. Technically, only one parcel is expanded at a time and the inner HTML isn't rendered, but it's still not great. What do you think about adding the parcel number, e.g., "owner-info-1"?

Reply:
I would use an Angular mechanism for getting the element instead of plain JS.
Check this post for a good example: https://stackoverflow.com/a/54418622

Reply:
That's way better. I was trying to avoid re-adding existing functionality, so reused `scrollToElement` from `portal-frontend/src/app/shared/utils/scroll-helper.ts`.